New release broke gvariant feature

If you build zbus from git using `cargo build --features gvariant` it would fail with the following 
```
user@toolbox ~/P/zbus (main)> cargo build --features gvariant
    Blocking waiting for file lock on build directory
   Compiling zvariant v5.1.0 (/home/user/Projects/zbus/zvariant)
error[E0004]: non-exhaustive patterns: `Format::GVariant` not covered
   --> zvariant/src/serialized/data.rs:234:28
    |
234 |         let mut de = match self.context.format() {
    |                            ^^^^^^^^^^^^^^^^^^^^^ pattern `Format::GVariant` not covered
    |
note: `Format` defined here
   --> /home/user/Projects/zbus/zvariant_utils/src/serialized.rs:5:1
    |
5   | pub enum Format {
    | ^^^^^^^^^^^^^^^
...
12  |     GVariant,
    |     -------- not covered
    = note: the matched value is of type `Format`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
    |
267 ~             .map(Deserializer::DBus)?,
268 ~             Format::GVariant => todo!(),
    |

error[E0004]: non-exhaustive patterns: `Format::GVariant` not covered
   --> zvariant/src/serialized/data.rs:307:28
    |
307 |         let mut de = match self.context.format() {
    |                            ^^^^^^^^^^^^^^^^^^^^^ pattern `Format::GVariant` not covered
    |
note: `Format` defined here
   --> /home/user/Projects/zbus/zvariant_utils/src/serialized.rs:5:1
    |
5   | pub enum Format {
    | ^^^^^^^^^^^^^^^
...
12  |     GVariant,
    |     -------- not covered
    = note: the matched value is of type `Format`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
    |
340 ~             .map(Deserializer::DBus)?,
341 ~             Format::GVariant => todo!(),
    |

error[E0004]: non-exhaustive patterns: `Format::GVariant` not covered
  --> zvariant/src/ser.rs:58:21
   |
58 |     let len = match ctxt.format() {
   |                     ^^^^^^^^^^^^^ pattern `Format::GVariant` not covered
   |
note: `Format` defined here
  --> /home/user/Projects/zbus/zvariant_utils/src/serialized.rs:5:1
   |
5  | pub enum Format {
   | ^^^^^^^^^^^^^^^
...
12 |     GVariant,
   |     -------- not covered
   = note: the matched value is of type `Format`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
   |
69 ~         },
70 +         Format::GVariant => todo!()
   |

error[E0004]: non-exhaustive patterns: `Format::GVariant` not covered
   --> zvariant/src/ser.rs:172:21
    |
172 |     let len = match ctxt.format() {
    |                     ^^^^^^^^^^^^^ pattern `Format::GVariant` not covered
    |
note: `Format` defined here
   --> /home/user/Projects/zbus/zvariant_utils/src/serialized.rs:5:1
    |
5   | pub enum Format {
    | ^^^^^^^^^^^^^^^
...
12  |     GVariant,
    |     -------- not covered
    = note: the matched value is of type `Format`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
    |
183 ~         },
184 +         Format::GVariant => todo!()
    |

error[E0004]: non-exhaustive patterns: `&zvariant_utils::signature::Signature::Maybe(_)` not covered
   --> zvariant/src/de.rs:198:11
    |
198 |     match signature {
    |           ^^^^^^^^^ pattern `&zvariant_utils::signature::Signature::Maybe(_)` not covered
    |
note: `zvariant_utils::signature::Signature` defined here
   --> /home/user/Projects/zbus/zvariant_utils/src/signature/mod.rs:42:1
    |
42  | pub enum Signature {
    | ^^^^^^^^^^^^^^^^^^
...
99  |     Maybe(Child),
    |     ----- not covered
    = note: the matched value is of type `&zvariant_utils::signature::Signature`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern or an explicit pattern as shown
    |
217 ~         Signature::Structure { .. } => de.deserialize_seq(visitor),
218 ~         &zvariant_utils::signature::Signature::Maybe(_) => todo!(),
    |

```

It is something I have noticed from the CI bump of zbus at https://github.com/bilelmoussaoui/oo7/pull/154.


Note this happens even if I have 
```
zvariant = { version = "5.1", default-features = false, features = ["gvariant"]}
```

--- a/zbus_macros/Cargo.toml
+++ b/zbus_macros/Cargo.toml
@@ -20,6 +20,7 @@ readme = "README.md"
 default = []
 # Enable blocking API.
 blocking-api = ["zbus/blocking-api"]
+gvariant = ["zvariant/gvariant", "zvariant_utils/gvariant"]
 
 [lib]
 proc-macro = true

